fhttp/http_client.go

```diff
@@ -513,12 +517,25 @@ func (c *Client) StreamFetch(ctx context.Context) (int, int64, uint) {
 		for strings.Contains(body, uuidToken) {
 			body = strings.Replace(body, uuidToken, generateUUID(), 1)
 		}
+		c.body = []byte(body)
 		bodyBytes := []byte(body)
 		req.ContentLength = int64(len(bodyBytes))
 		req.Body = io.NopCloser(bytes.NewReader(bodyBytes))
 	} else if len(c.body) > 0 {
 		req.Body = io.NopCloser(bytes.NewReader(c.body))
 	}
+
+	if c.bodyContainsNOWUTC {
+		nowUTC := nowFn().UTC()
+		body := string(c.body)
+		for strings.Contains(body, nowUTCToken) {
+			body = strings.Replace(body, nowUTCToken, nowUTC.String(), 1)
```

Comment:
.String() once, outside of the loop

but why not replaceall

Reply:
maybe uuid can use fixing too :) that also was contributed and I might have missed it in the then review

Reply:
hahah i'll look into it. I think I remember seeing uuid are unique.. so if you had a payload with multiple, they'd be all different..